squad/core/models.py

```diff
@@ -810,9 +815,6 @@ def history(self):
         self.__history__ = Test.History(since, count, last_different)
         return self.__history__
 
-    class Meta:
-        ordering = ['name']
```

Comment:
this will cause all the places where tests were ordered by name to not be anymore. maybe this will have annoying effects. or maybe ordering on `metadata__name` is too expensive for us to even consider?